### DISCUSSION TITLE
Tversky coefficient for rdkit cartridge

### DISCUSSION THREAD

**Discussion by annemaiw** [rdkit/rdkit#5847] [Q&A] [answered]
I would like to run a Tversky similarity search against the rdkit cartridge that I installed. For setting up the cartridge, I followed the instructions found here: https://www.rdkit.org/docs/Cartridge.html

Now I would like to create a function similar to _get_mfp2_neighbors_ but using Tversky instead of Tanimoto similarity. Would anybody know how to do this?

**Comment by ptosco** [comment, marked as answer]
@annemaiw You should only need to replace `tanimoto_sml` with `tversky_sml`.
The `tversky_sml` function takes the same parameters as `tanimoto_sml`, plus two float parameters which are Tversky's &alpha; and &beta; parameters:
```
tversky_sml(fp1, fp2, a, b)
```

**Reply by annemaiw** [reply]
Thank you, I had replaced tanimoto_sml with tversky_sml but forgotten about the two float parameters. Makes perfect sense!
